LiteCore/Storage/SQLiteDataFile.cc

```diff
@@ -366,7 +366,7 @@ path.path().c_str());
     void SQLiteDataFile::_beginTransaction(Transaction*) {
         checkOpen();
         LogVerbose(SQL, "BEGIN");
```

Comment:
While you're making changes, could just delete this log message? I just noticed it's redundant, since `_exec` will log. Thx!